DependencyInjection/CmfSeoExtension.php

```diff
@@ -79,20 +69,19 @@ public function load(array $configs, ContainerBuilder $container)
             $this->loadSonataAdmin($config['sonata_admin_extension'], $loader, $container, $sonataBundles);
         }
 
-        if ($config['enable_content_listener']) {
-
-            $loader->load('content-listener.xml');
-
-            if ($this->isConfigEnabled($container, $config['alternate_locale'])) {
-                $this->loadAlternateLocaleProvider($config['alternate_locale'], $container);
-            }
-        }
-
         $errorConfig = isset($config['error']) ? $config['error'] : array();
         $this->loadErrorHandling($errorConfig, $container);
 
         if ($this->isConfigEnabled($container, $config['sitemap'])) {
-            $this->loadSitemapHandling($config['sitemap'], $loader, $container);
+            $this->loadSitemapHandling($config['sitemap'], $loader, $container, $config['persistence']);
+        }
+
+        if ($this->isConfigEnabled($container, $config['content_listener'])) {
+            $this->loadContentListener($config['content_listener'], $loader, $container);
+        }
+
+        if ($this->isConfigEnabled($container, $config['alternate_locale'])) {
```

Comment:
moving this out of enable content listener, as the alternate locale provider is also used by sitemap.